src/applications/simple-forms/26-4555/definitions/customProfileAddress.js

```diff
@@ -0,0 +1,387 @@
+/**
```

Comment:
PR Looks great overall. Nice job.  besides double checking on the deleted sah/sahg that Nick/Gerrett mentioned,
could you also mention what is different in this file from its original definitions/profileAddress?  I see it's a lot copied over from there so it's hard to see what is different.

Reply:
@rhasselle-oddball @ndsprinkle , FYI, the _only_ difference is the AdditionalInformation component.  Displays in the expected position [under the <label>].  For some reason, Platform's profileAddress doesn't display that as expected.

Reply:
My "solution" was to change the actual definition in the forms-system directory. I added a `ui:order` property to force the additional info expander to always appear directly under the checkbox. I also updated the military base info text to use `va-additional-info` instead of just being text.